schemas/yaml/Sense.yml

```diff
@@ -150,6 +150,16 @@ properties:
     type:  string
     description: The scientific name for this item
 
+  semanticDomains:
+    title:       Semantic Domains
+    type:        array
+    uniqueItems: true
+    description: A list of the semantic domains this sense belongs to. Examples include color, body part, clothing, etc. The colon character can be used to represent subdomains e.g. quality:color:primary, color:red, etc.
```

Comment:
I like your more concise description than the one I wrote in the issue. This is much more easily readable.